workos/sso.py

```diff
@@ -103,6 +103,9 @@ def get_authorization_url(
         if state is not None:
             params["state"] = state
 
+        if redirect_uri is None:
+            raise ValueError("Incomplete arguments. Need to specify a Redirect URI")
```

Comment:
We should probably reference the actual parameter name, like we do for the other `ValueError` cases we raise:

```suggestion
            raise ValueError("Incomplete arguments. Need to specify a 'redirect_uri'.")
```